src/expandableCalendar/index.tsx

```diff
@@ -255,7 +282,7 @@ class ExpandableCalendar extends Component<Props, State> {
     if (!this.props.horizontal) {
       return Math.max(commons.screenHeight, commons.screenWidth);
     }
-    return CLOSED_HEIGHT + WEEK_HEIGHT * (this.numberOfWeeks - 1) + (this.props.hideKnob ? 12 : KNOB_CONTAINER_HEIGHT);
+    return this.closedHeight + WEEK_HEIGHT * (this.numberOfWeeks - 1) + (this.props.hideKnob ? 12 : KNOB_CONTAINER_HEIGHT);
```

Comment:
Since we are including a week row in the closed state height, we need to subtract 2 weeks from the open state height:

```suggestion
    return this.closedHeight + WEEK_HEIGHT * (this.numberOfWeeks - 2) + (this.props.hideKnob ? 12 : KNOB_CONTAINER_HEIGHT);
```

Reply:
You also need to make sure that we handle if `CLOSED_HEIGHT` constant is used instead of the prop

Reply:
Just tested this and this is not correct, what is happening is the following;
The hardcoded week height is off by a ~2-3px. The formula then proceeds to repeat the extra 2-3px 5-6 times. This turns the 2-3px offset which is barely visible into 10-15px. This in the end is the root cause of the 'extra' height. 

I've added a weekHeight prop so we can modify this for custom themes.